### PR TITLE
Service Mesh Doc Defects Round -2

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -25,16 +25,32 @@ For more information about how to install Helm, see the Helm installation docume
...
 
-You require a Anypoint Service Mesh license to perform the installation. If you do not have a license, contact your MuleSoft representative.
+You require a Anypoint Service Mesh license to perform the installation. Ensure that the license you specify is in a digested format. If you do not have a license, contact your MuleSoft representative.
+
+Note that the license you procure must contain the following entitlements:
+
+* API Gateway: api-gateway
+* Cluster: clustering
+
+*Need to add more infomation about the digested format.--Tomas*
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: For the time being, we will leave it as is.

File: modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc [hunk 2]
```diff
@@ -56,8 +60,8 @@ image:service-mesh-flows.jpeg[]
 The adapter communicates with Anypoint Platform asynchronously to get the latest policies and contracts, and eventually sends back the API analytics information.
 
 
+== See Also
 
-
-
-
+* https://istio.io/docs/reference/config/policy-and-telemetry/mixer-overview[Mixer Overview]
+* https://istio.io/docs/reference/config/policy-and-telemetry/mixer-overview/#adapters[Adapter Overview]
```
Comment (tbruno) [inline modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc]: Both links point to the same place?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 3]
```diff
@@ -47,6 +63,58 @@ If you provision the adapter with organization credentials, you will be able to
 If you specify environment credentials, you will only be able to manage APIs of that environment. MuleSoft recommends that you use the environment credentials instead of the organization credentials. For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster.
 
+
+== Configuring Your Environment Variable
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:65]: I would just put the instructions for linux based OS.

File: modules/ROOT/pages/service-mesh-faqs.adoc [hunk 4]
```diff
@@ -34,17 +30,17 @@ No, and this is not required. Apps deployed to Anypoint Runtime Fabric are alrea
 +
 Anypoint Service Mesh supports GKE (Google), AKS (Azure), and EKS (AWS)
 
-== Features Covered in GA Release
+// == Features Covered in GA Release
```
Comment (tbruno) [inline modules/ROOT/pages/service-mesh-faqs.adoc:33]: Thanks. You will delete all this, right?

**Review by tbruno — COMMENTED** [review]
Thanks for addressing feedback.

**Review by chiragnarangmulesoft — APPROVED** [review]

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (tbruno) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:65]: @chiragnarangmulesoft do you agree we shouldn't include all OS?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/service-mesh-faqs.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/service-mesh-faqs.adoc:33]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:65]: Chirag wanted instructions for all OSs. Done! Thanks!

**Pull request closed without merging**